apps/api/e2e/tests/competition.test.ts

```diff
@@ -4338,6 +4339,39 @@ describe("Competition API", () => {
       }
     });
 
+    test("rules endpoint should be publicly accessible", async () => {
+      // Setup: Create a competition (don't need to start it, just test the query parameter functionality)
+      const adminClient = createTestClient();
+      await adminClient.loginAsAdmin(adminApiKey);
+
+      const competitionName = `Public Rules Test ${Date.now()}`;
+      const competitionResponse = await adminClient.createCompetition({
+        name: competitionName,
+        description: "Test competition for public rules access",
+        tradingConstraints: {
+          minimumPairAgeHours: 24,
+          minimum24hVolumeUsd: 50000,
+          minimumLiquidityUsd: 100000,
+          minimumFdvUsd: 1000000,
+        },
+      });
+
+      expect(competitionResponse.success).toBe(true);
+      const competition = competitionResponse as CreateCompetitionResponse;
+
+      // Test with specific competition ID via query parameter (public access)
+      const specificResponse = await axios.get(
```

Comment:
we should just use the `adminClient.getCompetitionRules` method instead of axios